### PR TITLE
Portfolio Johan Löfgren

### PR DESCRIPTION
**Pull request by johlofg**
One thing I wonder is if I (as I did here) do some css to for exampel ul without giving it a class name. 
Should I place it somewhere special in the css file, top/bottom etc?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      <nav class="social-media-icons-header">
+        <a class="circle" href="#">
+          <img class="linkedin-white" src="./assets/linkedinVit.svg" alt="Link to my linkedin site">
+        </a>
+        <a class="circle" href="#">
+          <img class="github-white" src="./assets/githubVit.svg"  alt="Link to my Github site">
+        </a>
+        <a class="circle" href="#">
+          <img class="stack-white" src="./assets/stackVit.svg"  alt="Link to my stackoverflow site">
+        </a>
```
Comment (KarolCislowski) [inline index.html]: It would be nice to add some hover effects to make your visitor subconscious want to click this link 🧙🏻

File: index.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      </nav> 
+      <div class="profile-pic-container"> 
+        <img class="profile-picture" src="./assets/Studio_Project.png" alt="Picture of me">      
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="about-me-text">
+        <p tabindex="0">
+          <img class="start-icon" src="./assets/start-figure.svg" alt="A picture of a arrow in the start of the text">
+        ipsum dolor sit amet consectetur adipisicing elit. Qui iste asperiores maxime libero vitae, nihil perspiciatis voluptatem et nisi ab eveniet culpa velit! Provident dolores nihil praesentium ex, quo ab.</p>
```
Comment (KarolCislowski) [inline index.html]: This section is about you, so write a few words about yourself.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    </section>    
+    <section class="projects">  
+      <h3 class="heading" tabindex="0">FEATURED PROJECTS</h3>
+      <div class="featured-projects">
+        <div class="project-card-1"> 
+          <img class="project-img" src="assets/moviebot.PNG" alt="Image of project 1" tabindex="0">        
+          <span class="heading-on-pic">CHATBOT</span>
+          <div class="project-text" tabindex="0">
+            <h4>Movie chatbot</h4>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis molestias mollitia nobis, nulla magni
```
Comment (KarolCislowski) [inline index.html]: This paragraph should contain a short description of your project.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+          <span class="heading-on-pic">CHATBOT</span>
+          <div class="project-text" tabindex="0">
+            <h4>Movie chatbot</h4>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis molestias mollitia nobis, nulla magni        
+            </p>          
+          </div>
+          <div class="project-techniques-container">          
+            <span class="project-techniques" tabindex="0" >HTML5</span>
+            <span class="project-techniques" tabindex="0">CSS</span>
+            <span class="project-techniques" tabindex="0">JavaScript ES6</span>
```
Comment (KarolCislowski) [inline index.html]: this should be aligned to the left.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+        </div>
+    </section> 
+    <section class="my-thoughts-about-code">
+      <h3 class="heading" tabindex="0">MY THOUGHTS</h3>
+      <div class="article-with-img-container">    
+        <article class="article-with-img" tabindex="0">
+          <img class="my-thoughts-img" src="./assets/typewriter.jpg" alt="image related to my thoughts article">          
+          <div class="my-thoughts-text">
+              <h5 class="date">Jan 2020</h5><h4>title</h4>
+              <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum, nam dolor reiciendi Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum, nam dolor reiciendiLorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum, nam dolor
```
Comment (KarolCislowski) [inline index.html]: In this section, you need to add your own posts.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
You created a very nice template here. You need to fix some small issues. Then you must fill it with true content.

**Comment by KarolCislowski** [comment]
Think about some container to make all your content same width image

**Comment by johlofg** [comment]
Like a main that enclosure everything?

Den ons 7 apr. 2021 08:23Karol Cislowski ***@***.***> skrev:

**Comment by KarolCislowski** [comment]
If you create this container outside of your sections then your backgrounds will look odd. you need to make the container inside of your sections, align it to the centre and give it a max-width of 1200px or something like that

**Comment by johlofg** [comment]
Yeah thats true, could I just set the all the sections to a fixed width and
align them?

Den ons 7 apr. 2021 08:56Karol Cislowski ***@***.***> skrev:

**Comment by KarolCislowski** [comment]
You need to have sections with 100% width to make the background zebra trick, and only the content of sections needs to be shorter.

**Comment by johlofg** [comment]
Fair enough :) thx

Den ons 7 apr. 2021 09:13Karol Cislowski ***@***.***> skrev:

**Review by KarolCislowski — APPROVED** [review]
👍🏻

**Pull request closed without merging**